app/Helpers/helpers.php

```diff
@@ -0,0 +1,16 @@
+<?php
+function bannerImg()
+{
+    $context = stream_context_create(array('http' => array('header' => 'Connection: close\r\n')));
+    $url = 'https://viblo.asia/';
+    $start = 150;
+    $match_leng = 25000;
+    $content = file_get_contents($url, false, $context, $start, $match_leng);
+    $data = preg_match('/<div class="container px-0"><img src="(.*?)"/is', $content, $match);
+    if (isset($match[1])) {
+        return $match[1];
+    } else {
+        return null;
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.ClosingTag.NotAllowed</br>**Severity**: error</br>**File**: app/Helpers/helpers.php</br>**Line**: 16</br>A closing tag is not permitted at the end of a PHP file